packages/components/src/Layout/Layout.scss

```diff
@@ -6,6 +6,7 @@
 /// Layout height of the header
 /// @type Number (Unit)
 $tc-layout-header-height: $navbar-height;
+$tc-layout-footer-height: 2 * $padding-small + $font-size-small;
```

Comment:
This is dangereous since if we change padding or font-size to switch to `em`, `rem` unit, this will fails.
```scss
footer {
  flex-shrink: 0;
  flex-grow: 0;
} 
```
It could be enough?